packages/espressocash_app/lib/features/tokens/services/token_service.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:injectable/injectable.dart';
+import '../data/token_dto.dart';
+import '../data/token_repository.dart';
+
+@lazySingleton
+class TokenService {
+  TokenService(this._tokenRepository);
+
+  final TokenListRepository _tokenRepository;
+
+  Future<List<TokenDTO>> getAllTokens() => _tokenRepository.getAllTokens();
+
+  Stream<List<TokenDTO>> watchAllTokens() => _tokenRepository.watchAllTokens();
+
+  Future<void> insertToken(TokenDTO tokenDTO) =>
+      _tokenRepository.insertToken(tokenDTO);
+
+  Future<void> updateToken(TokenDTO tokenDTO) =>
+      _tokenRepository.updateToken(tokenDTO);
+
+  Future<void> deleteToken(TokenDTO tokenDTO) =>
+      _tokenRepository.deleteToken(tokenDTO);
```

Comment:
These are just proxy methods to the repository, doesn't make sense to have a service for that.